src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -353,6 +354,12 @@ class PrisonerIepLevelReviewService(
     }
   }
 
+  private fun getReviewCommentForEvent(prisonOffenderEvent: HMPPSDomainEvent) = when (prisonOffenderEvent.additionalInformation.reason) {
+    "NEW_ADMISSION", "READMISSION" -> "Default level assigned on arrival"
+    "TRANSFERRED" -> "Level transferred from previous establishment"
```

Comment:
at this point, we're choosing to not explain why the level may have been changed